### ISSUE TITLE
[4.0.0-beta.1] Error: Class 'Phalcon\Logger' not found

### ISSUE THREAD

**Issue by diplopito** [phalcon/cphalcon#14263]
### Expected and Actual Behavior

Expected: Use Phalcon\Logger to log a message.
Actual Behavior: Error: Class 'Phalcon\Logger' not found

Can be reproduced using the code in the documentation (and replacing the proper path for the log):

```php
<?php

use Phalcon\Logger;
use Phalcon\Logger\Adapter\Stream;

$adapter = new Stream('/logs/application.log');
$logger  = new Logger(
    'messages',
    [
        'main' => $adapter,
    ]
);

$logger->error('Something went wrong');
```

### Error in Windows:
Fatal error: Uncaught Error: Class 'Phalcon\Logger' not found in D:\xampp\htdocs\phalcon\logger.php:7 Stack trace: #0 {main} thrown in D:\xampp\htdocs\phalcon\logger.php on line 7

### Error in Linux
Fatal error:  Uncaught Error: Class 'Phalcon\\Logger' not found in /var/www/vhosts/xxx/yyy/logger.php:10\nStack trace:\n#0 {main}\n  thrown in /var/www/vhosts/xxx/yyy/logger.php on line 10

### Details

* Phalcon version: 4.0.0.-beta.1
* PHP Version: 7.2 & 7.3
* Operating System: Linux / Windows
* Installation type:  installing via package manager / dll
* Zephir version (if any):
* Server: Apache

**Comment by diplopito** [comment]
Current class is `Phalcon\Logger\Logger`; will be replaced by `Phalcon\Logger` in next Beta.

**Comment by ruudboon** [comment]
Looks like this isn't fixed in the beta.2 release

**Comment by sergeyklay** [comment]
Fixed in the `4.0.x` branch. Feel free to open a new issue if the problem appears again. Thank you the bug report.
